components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -443,30 +443,30 @@ paths:
         - Resource
     post:
       description: |
-        This API is used to store an attribute given by the user.
+        This API is used to create a new attribute for a given resource. 
       x-wso2-request: |
         curl -k -X POST "https://localhost:9443/api/identity/config-mgt/v1.0/resource/{resource-type}/{resource-name}" -H "accept: application/json" -H "Content-Type: application/json" -H "Authorization: Basic <Base64Encoded[username:password]>" -d "{"key": "string","value": "string"}""
       x-wso2-curl: |
         curl -k -X POST https://localhost:9443/api/identity/config-mgt/v1.0/resource/e-mail/smtp -H "accept: application/json" -H 'Content-Type: application/json' -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d '{"key": "from", "description": "abc.com"}'
       x-wso2-response: |
-        {"id": "42c8d3f1-48f9-40e7-98e2-2b6376731fc0","key": "from","value": "abc.com"}
+        {"key":"from","value":"abc.com"}
       summary: |
-        Create an attribute
+        Create attribute
       # post parameters:
       parameters:
         - name: resource-name
           in: path
-          description:  This represents the name of the attribute to be added.
+          description:  This represents the name of the attribute that is to be added.
           required: true
           type: 'string'
         - name: resource-type
           in: path
-          description:  This represents the type of the attribute to be added and can either be the name or id.
+          description:  This represents the type of attribute that is to be added and can either be the resource-type name or ID.
```

Comment:
This value can only be the name, not the id. Shall we change it accordingly?

Reply:
Fixed with [c2af0a2](https://github.com/wso2/carbon-identity-framework/commit/c2af0a29f9313baf5105a4898a9e26baceae00fa)